dotcom-rendering/src/web/components/NewsLetterSignupBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/react';
+import { from, until, brand, brandAlt } from '@guardian/source-foundations';
+
+import Logo from '../../static/logos/guardian-newsletters-logo.svg';
+
+const bannerStyle = css`
+	background-color: ${brand[400]};
+	color: ${brandAlt[400]};
+	padding: 10px 0;
+	font-weight: bold;
+`;
+
+const wrapperStyle = css`
+	padding-left: 16px;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+
+	${from.wide} {
+		width: 250px;
+	}
+
+	${until.wide} {
+		width: 170px;
+	}
+
+	${until.desktop} {
+		justify-content: flex-start;
+		width: 100%;
+	}
+`;
+
+export const NewsLetterSignupBanner = () => (
+	<div css={bannerStyle}>
```

Comment:
I think, now that [this change](https://github.com/guardian/dotcom-rendering/pull/4903) to `ContainerLayout` to enable a background colour just for the content has merged, this code could make use of that component here

Reply:
I refactored the static SVG for the 'newsletters' badge into a component and separated it from the main Guardian logo, so the `ContainerLayout` does work well for the banner now.